src/main/java/com/commercetools/sync/commons/utils/SphereClientConfigOptions.java

```diff
@@ -0,0 +1,107 @@
+package com.commercetools.sync.commons.utils;
+
+import io.sphere.sdk.client.SphereClientConfig;
+
+import java.util.concurrent.TimeUnit;
+
+public class SphereClientConfigOptions {
+    protected static final long DEFAULT_TIMEOUT = 30000;
+    protected static final long INITIAL_RETRY_DELAY = 200;
+    protected static final TimeUnit DEFAULT_TIMEOUT_TIME_UNIT = TimeUnit.MILLISECONDS;
+    protected static final int MAX_RETRIES = 5;
+    protected static final int MAX_PARALLEL_REQUESTS = 20;
+
+    private SphereClientConfig clientConfig;
+    protected long timeout;
+    protected long initialDelay;
+    protected TimeUnit timeUnit;
+    protected int maxRetries;
+    protected int maxParallelRequests;
```

Comment:
Can these variables be set as 'private'? I guess they can be accessed only by getter method.